docs/model_developers.rst

```diff
@@ -19,3 +19,4 @@ Model developer documentation
     model_developers/model_design.rst
     model_developers/resource_sets.rst
     model_developers/unmanaged_resources.rst
+    model_developers/dictpath.rst
```

Comment:
Why did you add this line here. This documentation is already referenced from the `Inmanta Lifecycle Service Manager` section. Or did you intend to move it because it's now part of inmanta-core and usable in a broader scope than only LSM?

Reply:
I moved it( [docs/lsm/allocation/dictpath.rst → docs/model_developers/dictpath.rst](https://github.com/inmanta/inmanta-core/pull/6656/files#diff-0b16210b9d9d3f9a439fefe95e294e3f834d54317e3e161777555e86e380ad79) )

indeed becauuse it now is usable in a broader scope. Isn't that a good idea?

Reply:
No. The move is fine. I just somehow overlooked the fact that the file was moved.

Reply:
I am not sure where is should come either. It's either here or we need to introduce a new section for this (like `utilities`). I am fine with both options.